cumulusci/tasks/salesforce/custom_settings.py

```diff
@@ -0,0 +1,124 @@
+import os
+import yaml
+
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.utils import os_friendly_path
+from cumulusci.core.exceptions import TaskOptionsError, CumulusCIException
+
+
+class UploadCustomSettings(BaseSalesforceApiTask):
```

Comment:
Consider calling it LoadCustomSettings for better symmetry with the extract/load terminology we use in the bulk data tasks.